pwndbg/gdblib/elf.py

```diff
@@ -415,7 +415,7 @@ def map_inner(ei_class, ehdr, objfile: str) -> Tuple[pwndbg.lib.memory.Page, ...
         vaddr = int(phdr.p_vaddr)
         offset = int(phdr.p_offset)
         flags = int(phdr.p_flags)
-        ptype = int(phdr.p_type)
+        # ptype = int(phdr.p_type)
```

Comment:
Why is this commented out instead of removed?

Reply:
I just left it out because I thought it might be useful in future. I was checking rest of flake8 warnings, should I remove this and other commented out lines in next PR?

Reply:
I think so, yes. If we ever need them we'll just add them back in, no need to keep commented code around.